tests/profiling/settings.js

```diff
@@ -1,11 +1,14 @@
 const {getAssets} = require('./dao.js');
+//const redisStore = require('ha-store-redis');
+const redisStore = require('../../../ha-redis-adapter');
```

Comment:
Is this intentional? I don't see the file in the project.

Reply:
I'm still troubleshooting an issue, this is a reference to a local `ha-store-redis` package.
I'll revert once I'm done.